curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/660d8ca387f989d6b25a3343.md

```diff
@@ -16,13 +16,13 @@ To see the new result, click on the `"Add New Task"` button. Then add a title an
 Your `taskObj` should have an `id` property.
 
 ```js
-assert.match(code, /const\s+taskObj\s*=\s*\{\s*id/)
+assert.match(code, /const\s+taskObj\s*=\s*\{\s*("|'|)id\1/)
 ```
 
-Your `id` property should have the value of the `titleInput.value`.
+You should have a `taskObj` object with an `id` property that have the value of the `titleInput.value`.
```

Comment:
```suggestion
The `id` property of your `taskObj` object should be set to `titleInput.value`.
```

Reply:
Hi @Ksound22, the updated hint doesn't tell user their object is invalid if it is the case

Reply:
@huyenltnguyen Line 16 only checks `id`, not the object itself (if it is properly closed by a curly bracket). I updated the regex to check the object, so I think it's needed to update the hint

Reply:
I've updated the hint as @Ksound22 suggestion